docs/main/user-guides/local-storage-user-guide/local-pv-hostpath/advanced-operations/xfs-quota/enable-xfs-quota.md

```diff
@@ -0,0 +1,183 @@
+---
+id: enable-xfs-quota
+title: Enable XFS Quota on LocalPV Hostpath
+keywords:
+ - OpenEBS LocalPV Hostpath Enable XFS Quota
+ - XFS Quota
+ - Enable XFS Quota
+ - Advanced Operations
+description: This section describes about enabling XFS quotas for OpenEBS LocalPV Hostpath. 
+---
+
+# Enable XFS Quota on LocalPV Hostpath
+
+This document provides the necessary steps to enable and configure XFS Quota on OpenEBS LocalPV HostPath. By following these instructions, you will install the OpenEBS LocalPV provisioner, create a StorageClass with XFS Quota support, and set up a PersistentVolumeClaim (PVC) to apply project quotas on the local volumes. It also includes the process for mounting the volume to an application pod and verifying that the quota is successfully applied.
+
+## Install the OpenEBS Dynamic LocalPV Provisioner
+
+1. To install the OpenEBS LocalPV Hostpath Provisioner, execute the following command:
+
+```
+kubectl apply -f https://openebs.github.io/charts/openebs-operator-lite.yaml
```

Comment:
We shouldn't recommend the operator yaml installation. Only install via helm chart by enabling only hostpath.